rubicon/objc/types.py

```diff
@@ -509,12 +554,8 @@ def compound_value_for_sequence(seq, tp):
 register_preferred_encoding(b'v', None)
 register_preferred_encoding(b'B', c_bool)
 
-# Note, we prefer to map b'c' to c_byte rather than c_char, because otherwise
-# ctypes converts the value into a one-character string which is generally
-# not what we want at all, especially when the b'c' represents a BOOL var.
 register_preferred_encoding(b'c', c_byte)
 register_preferred_encoding(b'C', c_ubyte)
-register_encoding(b'c', c_char)
```

Comment:
It's not clear why this is being removed...

Reply:
This isn't visible in the diff, but the same registration call is repeated a few lines below, so I removed this one because it's redundant. https://github.com/dgelessus/rubicon-objc/blob/d2886bdd22d58ca6bb49ca46b00722677f22d262/rubicon/objc/types.py#L580-L585